.travis.yml

```diff
@@ -7,18 +7,18 @@ stages:
 - deploy
 cache:
   pip: true
-  
+
 install:
 - pip install --upgrade pip
 - pip install poetry
-- poetry install
+- poetry install -E django
```

Comment:
I wonder if we really want to make `django` optional? What's the downside to always installing and testing it?

Reply:
None really, except conceptually it didn't seem like you should NEED django to work on this module, and it's always a bit more to install.

Reply:
I just imagined I'm an outside contributor and wanted to fix an unrelated bug. I'd be surprised that I need to install django for that. django alone pulls four packages; now imagine if we had more of these integrations.

Reply:
I think it should be required, and that the tests in `test_django` should always run. Otherwise, you won't find out that you've broken something in the `django` integration until your Travis build runs

Reply:
Ok, I'll keep the importorskip though in case we want to make it optional after all.